manpages.go

```diff
@@ -0,0 +1,32 @@
+package main
+
+import (
+	"fmt"
+	"regexp"
+	"strconv"
+	"strings"
+)
+
+var manpagesMatcher = regexp.MustCompile(`\b(\w+)\((\d+)\)(\W|$)`)
```

Comment:
Is there an example of a section not matching "\d[a-z]+"? I tried to find the most restrictive pattern that still matches everything important to minimize false-positives.